code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -84,6 +86,9 @@
 
 /obj/machinery/atmospherics/components/unary/thermomachine/RefreshParts()
 	. = ..()
+	thermomachine_refresh_parts()
+
+/obj/machinery/atmospherics/components/unary/thermomachine/proc/thermomachine_refresh_parts()
```

Comment:
Document this (three slash comment)

Reply:
Documented but I am not sure if it's exactly what's wanted so if I need to change it please let me know.